### PR TITLE
Configure ecr expiration policy

### PR DESCRIPTION
**Pull request by limed**
See SE-677

Expires by default after 1 year

### CODE REVIEW

**Comment by the-smooth-operator** [comment]
@limed thanks for doing this. I have just a question, why only expiring untagged images? `"tagStatus": "untagged",`
Also, I'd change the expiration time to 180 days, but I can do that myself.

File: terraform/deployment.tf [hunk 1]
```diff
@@ -279,6 +279,30 @@ EOF
...
+  repository = aws_ecr_repository.discourse.name
+
+  policy = <<EOF
+{
+  "rules": [
+    {
+      "rulePriority": 1,
+      "description": "Expire untagged image older than ${var.ecr_expire_days} days",
+      "selection": {
+        "tagStatus": "untagged",
```
Comment (the-smooth-operator) [inline terraform/deployment.tf]: can we change "untagged" for "any"?

**Review by the-smooth-operator — CHANGES_REQUESTED** [review]

File: terraform/deployment.tf (hunk 1 above)
Comment (limed) [reply terraform/deployment.tf]: Resolved in 49cf9d794c22876763850b0538f47fb3dca49f09

**Comment by the-smooth-operator** [comment]
thanks Ed!

**Pull request merged**